File: src/Author.ts
class Author {
  id: Number;
  firstName: String;
  lastName: String;
  email: String;
  get getFirstName(): String {
    return this.firstName;
  }
  set setFirstName(value: String) {
    this.firstName = value;
  }
  get getLastName(): String {
    return this.lastName;
  }
  set setLastName(value: String) {
    this.lastName = value;
  }
  get getEmail(): String {
    return this.email;
  }
  set setEmail(value: String) {
    this.email = value;
  }

  constructor(firstName: String, lastName: String) {
    this.firstName = firstName;
    this.lastName = lastName;
  }
}

File: webapi5/models/Author.cs

using System.Collections.Generic;

namespace webapi5.models
{
	public class Author
	{
		public int AuthorId { get; set; }
		public string FirstName { get; set; }
		public string LastName { get; set; }
		public List<Email> Emails { get; set; }
	}

}